npm/pkg/controlplane/controllers/v1/translatePolicy_test.go

```diff
@@ -1124,7 +1124,7 @@ func readPolicyYaml(policyYaml string) (*networkingv1.NetworkPolicy, error) {
 }
 
 func TestDenyAllPolicy(t *testing.T) {
-	denyAllPolicy, err := readPolicyYaml("testpolicies/deny-all-policy.yaml")
+	denyAllPolicy, err := readPolicyYaml("../../../../testpolicies/deny-all-policy.yaml")
```

Comment:
May be good to define directory as constant. So, next time, we do not change directory location in every code.
For example, 
```go
testPolicyDir := "../.../../../"
```
Then use testPolicyDir in all `readPolicyYaml` function